integration/build_test.go

```diff
@@ -151,7 +152,13 @@ func checkImageExists(t *testing.T, image string) {
 		return
 	}
 
-	client, err := docker.NewAPIClient(&runcontext.RunContext{})
+	cfg, err := kubectx.CurrentConfig()
+	failNowIfError(t, err)
+
+	// TODO: use the proper RunContext
```

Comment:
why can't we just use `runcontext.GetRunContext()` here?

Reply:
We don't really have SkaffoldOptions and SkaffoldPipeline available.